client/packages/portal-client/src/components/portal-router/PortalRouter.tsx

```diff
@@ -20,13 +20,35 @@ const routes: RouteObject[] = [
 				errorElement: <PortalMessagePage title="Fail to load view page" type={'Error'} />,
 			},
 			{
-				path: '/context-page/*',
-				element: <ContextPage />,
+				path: '/project',
+				element: <ViewPage />,
+				errorElement: <PortalMessagePage title="Fail to load view page" type={'Error'} />,
+			},
+			{
+				path: '/project/:contextId',
+				element: <ProjectPage />,
 				errorElement: <PortalMessagePage title="Fail to load context page" type={'Error'} />,
 			},
 			{
-				path: `/apps/:appKey/*`,
+				path: '/apps/:appKey',
 				element: <AppLoader />,
+				children: [
+					{
+						path: ':contextId/*',
+						element: <AppLoader />,
+						errorElement: <PortalMessagePage title="Fail to load application" type={'Error'} />,
+					},
+				],
+				errorElement: <PortalMessagePage title="Fail to load application" type={'Error'} />,
+			},
+			{
+				path: `/aka/*`,

```

Comment:
Why is aka/* a route?

Reply:
Im not sure if this is the right one  but there will be a rout used for applications to link to other application. Fusion Clasic has an aka route, but im not use if it's aka of go_to we are going to use.